command/pr_create.go

```diff
@@ -196,8 +193,15 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 
 		fmt.Fprintln(cmd.OutOrStdout(), pr.URL)
 	} else if action == PreviewAction {
+		var title, body string
```

Comment:
this is shadowing the existing `title` and `body` variables, breaking the preview in browser flow. unfortunately the tests didn't catch it.